[Review needed] Create 03-2021-CW21LT.md (#1769)
alan-turing-institute/the-turing-way: conferences/abstracts/03-2021-CW21LT.md

# Submitting a Lightning Talk Abstract on the Activism Chapter in the Ethics Book to SSIs Collaborations Workshop 2021 

## Lightning talk title
*For example, the name of the project, initiative or topic you want to present. 
If you want to use a lightning talk slot to introduce yourself and your work, then the title can simply be your full name!*

Introducing the Activism Chapter of the _Turing Way Guide to Ethical Research_. 

## Lightning talk abstract
*Just 2-3 sentences to add context to your lightning talk title in Question 4. This will not be published anywhere, but will help us to review your submission.*

There are many different scenarios where scientists and researchers would engage in activism to achieve change, such as increasing ethical standards of research, improving working conditions, or tackling systemic inequalities. 

> Scientists may engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or on topics unrelated to their career.

Reviewer: ```suggestion
```

> However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin, which the latest chapter of the _Turing Way Guide to Ethical Research_ seeks to offer.

Reviewer: ```suggestion
However, there are limited guides accessible to those who would like to achieve change but are unsure where to begin
The latest chapter of the _Turing Way Guide to Ethical Research_ aims to fill that gap.
The Activism chapter aims to help scientists engage in activism across topics related to their work, institutions, fields, cross-disciplinary issues such as open research, or even on topics unrelated to their career.
```

> This chapter welcomes additional contributions and case studies as it continues to expand to include examples of data science and tech activism from across the globe. 

Reviewer: ```suggestion
This chapter welcomes additional contributions and case studies as it continues to expand to support activism in data science and tech across the globe. 
```
